src/Traits/AssertionsTrait.php

```diff
@@ -18,36 +18,35 @@ trait AssertionsTrait
      * @param array  $actual
      * @param string $message
      */
-    protected function assertArrayValuesEquals(array $expected, array $actual, $message = '')
+    protected function assertArrayValuesEquals(array $expected, array $actual, string $message = ''): void
     {
         $this->assertEqualsCanonicalizing($expected, $actual, $message);
     }
 
     /**
-     * @param string $view_name
-     * @param string $message
+     * @param string      $viewName
+     * @param string|null $message
      */
-    public function assertViewExists(string $view_name, string $message = 'The view %s was not found.'): void
+    public function assertViewExists(string $viewName, ?string $message = null): void
     {
+        $message = $message ?: sprintf('The view %s was not found.', $viewName);
```

Comment:
This should only be done in the `catch`, because it's not used otherwise.

Reply:
Fixed.